framework/src/main/kotlin/br/com/zup/beagle/operations/builtin/Operations.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.operations.builtin
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/6a35e23efdaf4bacd5c464386e37eba1c8d8425c/framework/src/main/kotlin/br/com/zup/beagle/operations/builtin/Operations.kt#L17"></span>File '/Users/runner/work/beagle-backend-kotlin/beagle-backend-kotlin/framework/src/main/kotlin/br/com/zup/beagle/operations/builtin/Operations.kt' with '31' functions detected. Defined threshold inside files is set to '13'</td>
    </tr>
  </tbody>
</table>